qabelbox/src/main/java/de/qabel/core/repositories/AndroidClientDatabase.kt

```diff
@@ -33,8 +34,10 @@ class AndroidClientDatabase(connection: Connection) : AbstractClientDatabase(con
 
     @Throws(SQLException::class)
     override fun prepare(sql: String): PreparedStatement {
-        logger.trace(sql)
         if (connection is SQLDroidConnection) {
+            if(BuildConfig.DEBUG){
```

Comment:
Let the AndroidClientDatabase implement the AnkoLogger interface and use debug(sql) instead.